module2/homework/generateSequence/generateSequence.hpp

```diff
@@ -2,6 +2,15 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    if (count <= 0) {
+        return {};
+    }
+
+    std::vector<int> result;
+    result.reserve(count);
```

Comment:
You can define size of vector in constructor :)
```suggestion
    std::vector<int> result(count);

```

Reply:
> You can define size of vector in constructor :)

This change is not work. Do not passed tests.

Reply:
You are right, my fault :)